src/OpenLoco/GameCommands/ChangeCompanyFace.cpp

```diff
@@ -0,0 +1,87 @@
+#include "../CompanyManager.h"
+#include "../Interop/Interop.hpp"
+#include "../Localisation/StringIds.h"
+#include "../Objects/CompetitorObject.h"
+#include "../Objects/ObjectManager.h"
+#include "../Ui/WindowManager.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x00435506
+    static uint32_t changeCompanyFace(uint8_t flags, CompanyId targetCompanyId, ObjectHeader& targetHeader)
+    {
+        GameCommands::setExpenditureType(ExpenditureType::Miscellaneous);
```

Comment:
It also sets the error location to null. `GameCommands::setPosition({ static_cast<int16_t>(0x8000), 0, 0 });`